src/ResourceGraph/ResourceGraph/ChangeLog.md

```diff
@@ -17,4 +17,10 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Upcoming Release
\ No newline at end of file
+## Upcoming Release
+
+## Version 0.7.2
```

Comment:
@demyanenko you can remove this `## Version 0.7.2` header since we'll do the versioning later. The line below will go under the `## Upcoming Release` header. The content you have below with `## Version 0.7.1` and the line below that can stay.